apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/test/java/gov/cms/bfd/pipeline/rda/grpc/source/GrpcRdaSourceTest.java

```diff
@@ -107,11 +105,11 @@ public class GrpcRdaSourceTest {
    */
   @BeforeEach
   public void setUp() throws Exception {
-    MockitoAnnotations.openMocks(this);
+    closeable = MockitoAnnotations.openMocks(this);
```

Comment:
If we need to track a closeable for the mockito mocks can we just use a Rule instead?  I didn't even know there was a closeable associated with openMocks.